x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.ts

```diff
@@ -294,6 +294,7 @@ export const RelevanceTuningLogic = kea<
       actions.setResultsLoading(true);
 
       const filteredBoosts = removeEmptyValueBoosts(boosts);
+      const precisionSettings = values.isPrecisionTuningEnabled ? { precision } : {};
```

Comment:
Actual fix (thanks @afoucret !): Do not send precision when precision tuning is not enabled